bip-0001.mediawiki

```diff
@@ -81,6 +81,8 @@ Each BIP should have the following parts:
 
 * Reference Implementation -- The reference implementation must be completed before any BIP is given status "Final", but it need not be completed before the BIP is accepted. It is better to finish the specification and rationale first and reach consensus on it before writing code.
 
+* Copyright notice -- BIPs must contain a copyright notice placing them in the public domain.
```

Comment:
If you dont require public domain then you have to add an additional requirement of permissive licensing.